taker/src/main.rs

```diff
@@ -232,20 +257,18 @@ async fn main() -> Result<()> {
     let maker_identity = Identity::new(maker_id);
 
     let bitcoin_network = network.bitcoin_network();
-    let (ext_priv_key, identities, web_password) = match opts.umbrel_seed {
+    let (ext_priv_key, identities) = match opts.umbrel_seed {
         Some(seed_bytes) => {
             let seed = UmbrelSeed::from(seed_bytes);
             let ext_priv_key = seed.derive_extended_priv_key(bitcoin_network)?;
             let identities = seed.derive_identities();
-            let web_password = opts.password.unwrap_or_else(|| seed.derive_auth_password());
-            (ext_priv_key, identities, web_password)
+            (ext_priv_key, identities)
```

Comment:
🤔 Given that we remove `let web_password = opts.password.unwrap_or_else(|| seed.derive_auth_password());` is the password passed in compatible for existing installs?

Reply:
Do you mean the password  we pass in using `--password` or the one generated from the seed? 
The former one is compatible, the latter one is obsolete.

Reply:
I missed the part where we update the password in the db whenever it is passed with the parameter `--password` (line 388). To recap: If a password is provide we save it in the db. If no password provided we set a default on first startup an then ask the user to change it. That makes sense.

Reply:
Yes, exactly :)